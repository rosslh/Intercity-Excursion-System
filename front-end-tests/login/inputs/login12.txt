login agent
sellticket 12345 10
logout