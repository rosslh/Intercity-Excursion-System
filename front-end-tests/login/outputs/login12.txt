SEL 12345 10 00000 **** 0
EOS
