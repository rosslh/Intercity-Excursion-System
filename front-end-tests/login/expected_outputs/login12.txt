SEL 12345 10
EOS
